packages/modules/src/networkAnalysisDataVis/components/common/OptionSelector.tsx

```diff
@@ -35,14 +33,11 @@ export const OptionSelector = ({
   const classes = useStyles()
 
   const { setHighlightedDocItem } = useContext(NetworkGraphContext)
```

Comment:
This is probably unnecessary if you're removing documentation.

Reply:
Right. This part was not coming from `client-ui` so I just left it. Is it going to break anything if we leave it for now?